cirq-core/cirq/study/resolver.py

```diff
@@ -26,7 +26,8 @@
     import cirq
 
 
-ParamDictType = Dict['cirq.TParamKey', 'cirq.TParamVal']
+TParamValAny = Union['cirq.TParamVal', 'cirq.TParamValComplex']
```

Comment:
I don't think we need a TParamValAny since TParamValComplex (i.e. float, complex, sympy.Basic) is a supertype of TParamVal (i.e. float, sympy.Basic)

Reply:
Oh I didn't even notice that. Good catch!